src/services/Currency.js

```diff
@@ -0,0 +1,20 @@
+import { feathersClient } from '../lib/feathersClient';
+import { resolve } from 'dns';
+
+class Currency {
+  fiatTypes = ['BRL', 'CAD', 'CHF', 'CZK', 'EUR', 'GBP', 'MXN', 'THB', 'USD'];
```

Comment:
This needs to be array of objects { value, title } or you need to change how the fiat types are used.